lesson28/src/js/register.js

```diff
@@ -98,3 +99,17 @@ const isRelatedTarget = (event,target) => {
     const related = event.relatedTarget ? event.relatedTarget.id : "unknown";
     return related === target;
 }
+
+const saveUserData = () => {
+    const userName = document.querySelector('[data-id="user_name"]');
+    const email = document.querySelector('[data-id="email"]');
+    const password = document.querySelector('[data-id="new-password"]');
```

Comment:
nits
I thought it would be better to have a matching snake case and kebab case for data-id.

Reply:
That is correct!
I fixed.
https://github.com/nakagawamai/js-lesson/pull/38/commits/fb7015a88a6e07b5004102fcc71be7f52ceca84b